docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -156,13 +156,20 @@
                         others' checkpoints, and perhaps interfere with each other in other ways.
                     </td>
                 </tr>
-                <tr>
                     <td class="property" id="job-coordinator-system">job.coordinator.system</td>
                     <td class="default"></td>
                     <td class="description">
                         <strong>Required:</strong> The <span class="system">system-name</span> to use for creating and maintaining the <a href="../container/coordinator-stream.html">Coordinator Stream</a>.
                     </td>
                 </tr>
+                <tr>
+                    <td class="property" id="job-default-system">job.default.system</td>
```

Comment:
@navina For A, I misspoke. I didn't mean the job would ignore the old configs but that the user would. 

I don't understand your example for system scope. To use the exact once producer for metadata and the normal producer for others, one would do this. 

systems.kafka-exact.samza.factory=org.apache.pkg.KafkaExactOnceSystemFactory
systems.kafka-at-least.samza.factory=org.apache.pkg.KafkaAtLeastOnceSystemFactory
systems.hdfs.samza.factory=org.apache.pkg.HdfsSystemFactory

job.default.system=kafka-at-least
job.metadata.system=kafka-exact
streams.my-only-hdfs-stream.samza.system=hdfs

This demonstrates:
1. How to default all metadata streams to use one system
2. How to default all other streams to use another system
3. How to override the system for a particular stream